ios/brave-ios/Sources/BrowserMenu/BrowserMenu.swift

```diff
@@ -118,7 +119,7 @@ public struct BrowserMenu: View {
                 Spacer()
                 Text(region.flag)
                 Text(region.displayName)
-                if region.smartProxySupported {
+                if region.smartProxySupported, BraveVPN.isSmartProxyRoutingEnabled {
```

Comment:
Should be part of the view model, not accessed directly. Better to add it to the `VPNStatus` instead

Reply:
fixe in [1e6dd2f](https://github.com/brave/brave-core/pull/32598/commits/1e6dd2f065a6378955fa5cfffbaca1697eb12e02)